src/views/validated-designs/types.ts

```diff
@@ -9,7 +9,7 @@ export interface HvdCategoryGroup {
 	slug: string // start of slug e.g. terraform-operation-guides-adoption, used to add to it's guides after the category group is created
 	title: string // from YAML
 	description: string // from YAML
-	product: Exclude<ProductSlug, 'sentinel'>
+	productSlug: Exclude<ProductSlug, 'sentinel'>
```

Comment:
I _think_ this can be changed now that Sentinel has been migrated to dev portal
```suggestion
	productSlug: ProductSlug
```

Reply:
Unfortunately it's still listed as a product slug: https://github.com/hashicorp/dev-portal/blob/052e835c42b28a5959a3ee375aed5da8de2559c0/src/types/products.ts#L15

😢